package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Little Lyon Heart",
   "main": "src/index.js",
   "scripts": {
-    "build": "babel src -d lib",
+    "build": "babel src -d lib --copy-files",
     "dev": "nodemon --exec babel-node src/index.js",
     "forever:start": "forever start lib/index.js",
     "forever:stop": "forever stop lib/index.js || exit 0",
     "prestart": "npm run build && npm run forever:stop",
-    "start": "npm run forever:start"
+    "start": "npm run forever:start",
+    "test:lint": "eslint ./src && prettier --check ./src/**/*.js",
+    "test": "npm run test:lint"
```

Comment:
I think these should just be collapsed into `npm run lint`, these aren't really tests in the `npm test` sense, they're code styling checks

Reply:
I splitted the `test:lint` into `test:lint` and `test:prettier` so we have a better separation of concern. We can still put everything in the `test` script, but I believe we will soon have to implement unit tests, and having separated scripts will be for the best.
What do you think?

Reply:
Yeah, I think splitting them is good but instead of `test` they should be something like `lint`, `lint:syntax` and `lint:format` - `npm run test`/`npm test` should (eventually) be running the static/unit/integration/e2e tests, syntax and format checks are static analysis but they aren't tests